### PR TITLE
OpenSceneGraph: bump to 3.6.3

### PR DESCRIPTION
**Pull request by kenmays**
Built and tested on hrev52339 x64.

### CODE REVIEW

File: games-engines/openscenegraph/openscenegraph-3.6.2.recipe [hunk 1]
```diff
@@ -10,11 +10,11 @@ virtual reality industries."
...
 COPYRIGHT="2002-2018 Robert Osfield and others"
 LICENSE="OSGPL"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-$portVersion.tar.gz"
-CHECKSUM_SHA256="6f57134ea74a39f1c7b24c285e6278cf906c47f6c681573b86d12173a466efed"
+CHECKSUM_SHA256="762c6601f32a761c7a0556766097558f453f23b983dd75bcf90f922e2d077a34"
 SOURCE_DIR="OpenSceneGraph-OpenSceneGraph-$portVersion"
-PATCHES="openscenegraph-$portVersion.patchset"
+PATCHES="openscenegraph-3.6.0.patchset"
```
Comment (alaviss) [inline games-engines/openscenegraph/openscenegraph-3.6.2.recipe]: Please revert this change then copy the current `3.6.0` patch to `3.6.2`.

Comment (kenmays) [reply games-engines/openscenegraph/openscenegraph-3.6.2.recipe]: Done
